### PR TITLE
Feature/Testimonial: Add carousel component

### PR DESCRIPTION
**Pull request by AbigailFernandes**

### CODE REVIEW

File: src/components/Carousel.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Carousel from 'nuka-carousel';
+import home1 from '../img/home/home-1.jpg';
+import home2 from '../img/home/home-2.jpg';
+import home3 from '../img/home/home-3.jpg';
+import home4 from '../img/home/home-4.jpg';
+import home5 from '../img/home/home-5.jpg';
+const images = [
```
Comment (sanchitgn) [inline src/components/Carousel.js]: Add new line before this

Comment (sanchitnevgi) [inline src/components/Carousel.js]: Add new line before this

File: src/components/Carousel.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  { src: home1 },
+  { src: home2 },
+  { src: home3 },
+  { src: home4 },
+  { src: home5 },
+];
+const AppCarousel = () => (
```
Comment (sanchitnevgi) [inline src/components/Carousel.js]: New line before this

File: src/components/Carousel.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Carousel from 'nuka-carousel';
+import home1 from '../img/home/home-1.jpg';
+import home2 from '../img/home/home-2.jpg';
```
Comment (sanchitnevgi) [inline src/components/Carousel.js]: Use index.js in /img/home. Use import * from

File: src/components/Carousel.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+const images = [
+  { src: home1 },
+  { src: home2 },
+  { src: home3 },
+  { src: home4 },
+  { src: home5 },
+];
+const AppCarousel = () => (
+  <Carousel
+    // initialSlideHeight={300}
```
Comment (sanchitnevgi) [inline src/components/Carousel.js]: Remove commented code

File: src/pages/gallery.js [hunk 5]
```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import Gallery from '../components/Gallery';
 
 export default () => (
-  <section className='section'>
+  <section className="section">
```
Comment (sanchitnevgi) [inline src/pages/gallery.js:5]: Single quotes

File: src/components/Carousel.js (hunk 3 above)
Comment (AbigailFernandes) [reply src/components/Carousel.js]: done

File: src/components/Carousel.js (hunk 1 above)
Comment (AbigailFernandes) [reply src/components/Carousel.js]: done

File: src/components/Carousel.js (hunk 2 above)
Comment (AbigailFernandes) [reply src/components/Carousel.js]: done

**Review by sanchitnevgi — APPROVED** [review]

**Pull request merged**